From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
Date: Fri, 17 Apr 2020 23:44:35 +0200
Subject: [ARCHEOLOGY] Switch sunxi and sunxi64 to u-boot v2020.04 (#1894)

> X-Git-Archeology: > recovered message: > * Disable DE2 in u-boot to improve EDID detection
> X-Git-Archeology: > recovered message: > * Remove deprecated patches
> X-Git-Archeology: > recovered message: > * Define BOOTBRANCH in top level
> X-Git-Archeology: > recovered message: > Signed-off-by: Igor Pecovnik <igor.pecovnik@gmail.com>
> X-Git-Archeology: - Revision 5e8cbd8e6aaa87ee03d08c761d5e765e81c4c686: https://github.com/armbian/build/commit/5e8cbd8e6aaa87ee03d08c761d5e765e81c4c686
> X-Git-Archeology:   Date: Fri, 17 Apr 2020 23:44:35 +0200
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Switch sunxi and sunxi64 to u-boot v2020.04 (#1894)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 4ba494ae6ec0515c9e076624bb0ec7fa7f97655d: https://github.com/armbian/build/commit/4ba494ae6ec0515c9e076624bb0ec7fa7f97655d
> X-Git-Archeology:   Date: Sat, 19 Sep 2020 10:09:05 -0700
> X-Git-Archeology:   From: 5kft <5kft@users.noreply.github.com>
> X-Git-Archeology:   Subject: [ sunxi-dev ] move u-boot to v2020.07
> X-Git-Archeology:
> X-Git-Archeology: - Revision ce6783c07ff7a4b0e44bd69521253756ad08f368: https://github.com/armbian/build/commit/ce6783c07ff7a4b0e44bd69521253756ad08f368
> X-Git-Archeology:   Date: Thu, 15 Oct 2020 21:30:54 +0200
> X-Git-Archeology:   From: Igor Pecovnik <igor.pecovnik@gmail.com>
> X-Git-Archeology:   Subject: Adjust remaining Allwinner u-boot patches due to u-boot upgrade to 2020.10
> X-Git-Archeology:
> X-Git-Archeology: - Revision 21fc940aaf761bc65f8d81c6e68584757b43c8fb: https://github.com/armbian/build/commit/21fc940aaf761bc65f8d81c6e68584757b43c8fb
> X-Git-Archeology:   Date: Sun, 31 Jul 2022 10:34:48 +0200
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Several small bug fixes mainly outside supported areas (#4032)
> X-Git-Archeology:
> X-Git-Archeology: - Revision f6a09def35e647a5442e0f92f399485be29f19a9: https://github.com/armbian/build/commit/f6a09def35e647a5442e0f92f399485be29f19a9
> X-Git-Archeology:   Date: Thu, 10 Nov 2022 21:49:36 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Moving patches to per board and removing obsolete (#4409)
> X-Git-Archeology:
---
 configs/Mele_A1000_defconfig                  | 1 +
 configs/Orangepi_mini_defconfig               | 1 +
 configs/a64-olinuxino-emmc_defconfig          | 1 +
 configs/a64-olinuxino_defconfig               | 1 +
 configs/bananapi_m1_plus_defconfig            | 1 +
 configs/bananapi_m2_plus_h3_defconfig         | 1 +
 configs/bananapi_m2_plus_h5_defconfig         | 1 +
 configs/bananapi_m2_zero_defconfig            | 1 +
 configs/bananapi_m64_defconfig                | 1 +
 configs/libretech_all_h3_cc_h2_plus_defconfig | 1 +
 configs/libretech_all_h3_cc_h3_defconfig      | 1 +
 configs/libretech_all_h3_cc_h5_defconfig      | 1 +
 configs/libretech_all_h5_cc_h5_defconfig      | 1 +
 configs/nanopi_a64_defconfig                  | 1 +
 configs/nanopi_m1_defconfig                   | 1 +
 configs/orangepi_lite2_defconfig              | 1 +
 configs/orangepi_lite_defconfig               | 1 +
 configs/orangepi_one_defconfig                | 1 +
 configs/orangepi_one_plus_defconfig           | 1 +
 configs/orangepi_pc2_defconfig                | 1 +
 configs/orangepi_pc_defconfig                 | 1 +
 configs/orangepi_plus2e_defconfig             | 1 +
 configs/orangepi_plus_defconfig               | 1 +
 configs/orangepi_prime_defconfig              | 1 +
 configs/orangepi_win_defconfig                | 1 +
 configs/pine64-lts_defconfig                  | 1 +
 configs/pine64_plus_defconfig                 | 1 +
 configs/pine_h64_defconfig                    | 1 +
 28 files changed, 28 insertions(+)

diff --git a/configs/Mele_A1000_defconfig b/configs/Mele_A1000_defconfig
index d3a01275cf..94f76322d5 100644
--- a/configs/Mele_A1000_defconfig
+++ b/configs/Mele_A1000_defconfig
@@ -18,3 +18,4 @@ CONFIG_SUN4I_EMAC=y
 CONFIG_SCSI=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
+CONFIG_VIDEO_DE2=n
diff --git a/configs/Orangepi_mini_defconfig b/configs/Orangepi_mini_defconfig
index f8f3190405..001f26f6cb 100644
--- a/configs/Orangepi_mini_defconfig
+++ b/configs/Orangepi_mini_defconfig
@@ -27,3 +27,4 @@ CONFIG_SUN7I_GMAC=y
 CONFIG_SCSI=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
+CONFIG_VIDEO_DE2=n
diff --git a/configs/a64-olinuxino-emmc_defconfig b/configs/a64-olinuxino-emmc_defconfig
index 8ec9eb3e9c..4ccb4c8467 100644
--- a/configs/a64-olinuxino-emmc_defconfig
+++ b/configs/a64-olinuxino-emmc_defconfig
@@ -10,3 +10,4 @@ CONFIG_SUPPORT_EMMC_BOOT=y
 CONFIG_SUN8I_EMAC=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
+CONFIG_VIDEO_DE2=n
diff --git a/configs/a64-olinuxino_defconfig b/configs/a64-olinuxino_defconfig
index 99fa24ae23..d00bd9e7e3 100644
--- a/configs/a64-olinuxino_defconfig
+++ b/configs/a64-olinuxino_defconfig
@@ -11,3 +11,4 @@ CONFIG_DRAM_ZQ=3881949
 CONFIG_SUN8I_EMAC=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
+CONFIG_VIDEO_DE2=n
diff --git a/configs/bananapi_m1_plus_defconfig b/configs/bananapi_m1_plus_defconfig
index 2f0c22f62f..b3d68c62d1 100644
--- a/configs/bananapi_m1_plus_defconfig
+++ b/configs/bananapi_m1_plus_defconfig
@@ -23,3 +23,4 @@ CONFIG_SUN7I_GMAC=y
 CONFIG_SCSI=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
+CONFIG_VIDEO_DE2=n
diff --git a/configs/bananapi_m2_plus_h3_defconfig b/configs/bananapi_m2_plus_h3_defconfig
index 26ced59fb0..a723dc4847 100644
--- a/configs/bananapi_m2_plus_h3_defconfig
+++ b/configs/bananapi_m2_plus_h3_defconfig
@@ -11,3 +11,4 @@ CONFIG_SUN8I_EMAC=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
 CONFIG_USB_MUSB_GADGET=y
+CONFIG_VIDEO_DE2=n
diff --git a/configs/bananapi_m2_plus_h5_defconfig b/configs/bananapi_m2_plus_h5_defconfig
index fb6c945919..2e4ecdfbe8 100644
--- a/configs/bananapi_m2_plus_h5_defconfig
+++ b/configs/bananapi_m2_plus_h5_defconfig
@@ -11,3 +11,4 @@ CONFIG_SUN8I_EMAC=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
 CONFIG_USB_MUSB_GADGET=y
+CONFIG_VIDEO_DE2=n
diff --git a/configs/bananapi_m2_zero_defconfig b/configs/bananapi_m2_zero_defconfig
index ac3f8f5ab8..58ce3e170f 100644
--- a/configs/bananapi_m2_zero_defconfig
+++ b/configs/bananapi_m2_zero_defconfig
@@ -6,3 +6,4 @@ CONFIG_MACH_SUN8I_H3=y
 CONFIG_DRAM_CLK=408
 CONFIG_MMC0_CD_PIN=""
 # CONFIG_SYS_MALLOC_CLEAR_ON_INIT is not set
+CONFIG_VIDEO_DE2=n
diff --git a/configs/bananapi_m64_defconfig b/configs/bananapi_m64_defconfig
index 292044d7b8..0d47ca6415 100644
--- a/configs/bananapi_m64_defconfig
+++ b/configs/bananapi_m64_defconfig
@@ -11,3 +11,4 @@ CONFIG_SUN8I_EMAC=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
 CONFIG_USB_MUSB_GADGET=y
+CONFIG_VIDEO_DE2=n
diff --git a/configs/libretech_all_h3_cc_h2_plus_defconfig b/configs/libretech_all_h3_cc_h2_plus_defconfig
index 8725fe64cd..e2b58e47f5 100644
--- a/configs/libretech_all_h3_cc_h2_plus_defconfig
+++ b/configs/libretech_all_h3_cc_h2_plus_defconfig
@@ -9,3 +9,4 @@ CONFIG_MMC_SUNXI_SLOT_EXTRA=2
 CONFIG_SUN8I_EMAC=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
+CONFIG_VIDEO_DE2=n
diff --git a/configs/libretech_all_h3_cc_h3_defconfig b/configs/libretech_all_h3_cc_h3_defconfig
index 5275fdc36d..11cd4e0fe3 100644
--- a/configs/libretech_all_h3_cc_h3_defconfig
+++ b/configs/libretech_all_h3_cc_h3_defconfig
@@ -9,3 +9,4 @@ CONFIG_MMC_SUNXI_SLOT_EXTRA=2
 CONFIG_SUN8I_EMAC=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
+CONFIG_VIDEO_DE2=n
diff --git a/configs/libretech_all_h3_cc_h5_defconfig b/configs/libretech_all_h3_cc_h5_defconfig
index 9627401949..6fefb6756d 100644
--- a/configs/libretech_all_h3_cc_h5_defconfig
+++ b/configs/libretech_all_h3_cc_h5_defconfig
@@ -9,3 +9,4 @@ CONFIG_MMC_SUNXI_SLOT_EXTRA=2
 CONFIG_SUN8I_EMAC=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
+CONFIG_VIDEO_DE2=n
diff --git a/configs/libretech_all_h5_cc_h5_defconfig b/configs/libretech_all_h5_cc_h5_defconfig
index c3aa4b1061..55e4313961 100644
--- a/configs/libretech_all_h5_cc_h5_defconfig
+++ b/configs/libretech_all_h5_cc_h5_defconfig
@@ -12,3 +12,4 @@ CONFIG_SUN8I_EMAC=y
 CONFIG_SPI=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
+CONFIG_VIDEO_DE2=n
diff --git a/configs/nanopi_a64_defconfig b/configs/nanopi_a64_defconfig
index 70fc257eeb..3eb285ee98 100644
--- a/configs/nanopi_a64_defconfig
+++ b/configs/nanopi_a64_defconfig
@@ -8,3 +8,4 @@ CONFIG_RESERVE_ALLWINNER_BOOT0_HEADER=y
 CONFIG_SUN8I_EMAC=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
+CONFIG_VIDEO_DE2=n
diff --git a/configs/nanopi_m1_defconfig b/configs/nanopi_m1_defconfig
index dc2dbd6290..5abf34aa34 100644
--- a/configs/nanopi_m1_defconfig
+++ b/configs/nanopi_m1_defconfig
@@ -7,3 +7,4 @@ CONFIG_DRAM_CLK=408
 # CONFIG_SYS_MALLOC_CLEAR_ON_INIT is not set
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
+CONFIG_VIDEO_DE2=n
diff --git a/configs/orangepi_lite2_defconfig b/configs/orangepi_lite2_defconfig
index 75c97d6b89..389687a456 100644
--- a/configs/orangepi_lite2_defconfig
+++ b/configs/orangepi_lite2_defconfig
@@ -9,3 +9,4 @@ CONFIG_MMC0_CD_PIN="PF6"
 # CONFIG_SYS_MALLOC_CLEAR_ON_INIT is not set
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
+CONFIG_VIDEO_DE2=n
diff --git a/configs/orangepi_lite_defconfig b/configs/orangepi_lite_defconfig
index ab64319875..1a5ea7fe4d 100644
--- a/configs/orangepi_lite_defconfig
+++ b/configs/orangepi_lite_defconfig
@@ -9,3 +9,4 @@ CONFIG_DRAM_ODT_EN=y
 # CONFIG_SYS_MALLOC_CLEAR_ON_INIT is not set
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
+CONFIG_VIDEO_DE2=n
diff --git a/configs/orangepi_one_defconfig b/configs/orangepi_one_defconfig
index cbe2c97f2b..4572c76579 100644
--- a/configs/orangepi_one_defconfig
+++ b/configs/orangepi_one_defconfig
@@ -10,3 +10,4 @@ CONFIG_DRAM_ODT_EN=y
 CONFIG_SUN8I_EMAC=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
+CONFIG_VIDEO_DE2=n
diff --git a/configs/orangepi_one_plus_defconfig b/configs/orangepi_one_plus_defconfig
index 55a8b003fb..19db83a680 100644
--- a/configs/orangepi_one_plus_defconfig
+++ b/configs/orangepi_one_plus_defconfig
@@ -9,3 +9,4 @@ CONFIG_MMC0_CD_PIN="PF6"
 # CONFIG_SYS_MALLOC_CLEAR_ON_INIT is not set
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
+CONFIG_VIDEO_DE2=n
diff --git a/configs/orangepi_pc2_defconfig b/configs/orangepi_pc2_defconfig
index 5a7dbd45e8..cb8ca5d981 100644
--- a/configs/orangepi_pc2_defconfig
+++ b/configs/orangepi_pc2_defconfig
@@ -20,3 +20,4 @@ CONFIG_SPI=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
 CONFIG_USB_MUSB_GADGET=y
+CONFIG_VIDEO_DE2=n
diff --git a/configs/orangepi_pc_defconfig b/configs/orangepi_pc_defconfig
index 905ff7b127..4aacb83bab 100644
--- a/configs/orangepi_pc_defconfig
+++ b/configs/orangepi_pc_defconfig
@@ -12,3 +12,4 @@ CONFIG_SUN8I_EMAC=y
 CONFIG_SY8106A_POWER=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
+CONFIG_VIDEO_DE2=n
diff --git a/configs/orangepi_plus2e_defconfig b/configs/orangepi_plus2e_defconfig
index 879f01dcd8..96ce325066 100644
--- a/configs/orangepi_plus2e_defconfig
+++ b/configs/orangepi_plus2e_defconfig
@@ -16,3 +16,4 @@ CONFIG_SUN8I_EMAC=y
 CONFIG_SY8106A_POWER=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
+CONFIG_VIDEO_DE2=n
diff --git a/configs/orangepi_plus_defconfig b/configs/orangepi_plus_defconfig
index aa3f51e97b..4b2ac4e05c 100644
--- a/configs/orangepi_plus_defconfig
+++ b/configs/orangepi_plus_defconfig
@@ -18,3 +18,4 @@ CONFIG_SUN8I_EMAC=y
 CONFIG_SY8106A_POWER=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
+CONFIG_VIDEO_DE2=n
diff --git a/configs/orangepi_prime_defconfig b/configs/orangepi_prime_defconfig
index 551348f8c6..2e19c27942 100644
--- a/configs/orangepi_prime_defconfig
+++ b/configs/orangepi_prime_defconfig
@@ -12,3 +12,4 @@ CONFIG_SUN8I_EMAC=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
 CONFIG_USB_MUSB_GADGET=y
+CONFIG_VIDEO_DE2=n
diff --git a/configs/orangepi_win_defconfig b/configs/orangepi_win_defconfig
index 3b78ad7e52..eba4af1bda 100644
--- a/configs/orangepi_win_defconfig
+++ b/configs/orangepi_win_defconfig
@@ -13,3 +13,4 @@ CONFIG_SUN8I_EMAC=y
 CONFIG_SPI=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
+CONFIG_VIDEO_DE2=n
diff --git a/configs/pine64-lts_defconfig b/configs/pine64-lts_defconfig
index 45a9e77e0e..33a7cc723c 100644
--- a/configs/pine64-lts_defconfig
+++ b/configs/pine64-lts_defconfig
@@ -15,3 +15,4 @@ CONFIG_SUN8I_EMAC=y
 CONFIG_SPI=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
+CONFIG_VIDEO_DE2=n
diff --git a/configs/pine64_plus_defconfig b/configs/pine64_plus_defconfig
index f42f4e5923..400519cd89 100644
--- a/configs/pine64_plus_defconfig
+++ b/configs/pine64_plus_defconfig
@@ -11,3 +11,4 @@ CONFIG_PHY_REALTEK=y
 CONFIG_SUN8I_EMAC=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
+CONFIG_VIDEO_DE2=n
diff --git a/configs/pine_h64_defconfig b/configs/pine_h64_defconfig
index 1e730dd9fa..7e772a5a98 100644
--- a/configs/pine_h64_defconfig
+++ b/configs/pine_h64_defconfig
@@ -2,6 +2,7 @@ CONFIG_ARM=y
 CONFIG_ARCH_SUNXI=y
 CONFIG_DEFAULT_DEVICE_TREE="sun50i-h6-pine-h64"
 CONFIG_SPL=y
+CONFIG_VIDEO_DE2=n
 CONFIG_MACH_SUN50I_H6=y
 CONFIG_SUNXI_DRAM_H6_LPDDR3=y
 CONFIG_MACPWR="PC16"
-- 
Armbian

